ERCS/erc-tbd.md

```diff
@@ -0,0 +1,182 @@
+---
+number: TBD
```

Comment:
```suggestion
eip: 7731
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.